mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -66,8 +66,9 @@ CLASS cqn net/minecraft/client/MinecraftClient
 	FIELD ax blockColorMap Lcqw;
 	FIELD ay itemColorMap Lcqz;
 	FIELD az framebuffer Lcov;
-	FIELD b defaultFontRendererId Lqh;
-	FIELD c altFontRendererId Lqh;
+	FIELD b DEFAULT_TEXT_RENDERER_ID Lqh;
+	FIELD c ALT_TEXT_RENDERER_ID Lqh;
+	FIELD d voidFuture Ljava/util/concurrent/CompletableFuture;
```

Comment:
???

Reply:
`public static CompletableFuture<Void> voidFuture = CompletableFuture.completedFuture(Void.INSTANCE);`

Reply:
It's probably meant to be final too. It's not modified anywhere and `CompletableFuture.completedFuture(Void.INSTANCE);` is the only possible non-null value.

Reply:
Is it *used* anywhere? If not...

Reply:
Yes, it's used for methods that expect a non-null future parameter, such as: `this.a(resourceManager.reload(SystemUtil.e(), this, list2, voidFuture));`

Reply:
Ah, so that's how it is. Right.